packages/headless/src/controllers/facets/facet-search/specific/headless-facet-search.ts

```diff
@@ -46,7 +46,15 @@ export function buildFacetSearch(
     },
 
     get state() {
-      return genericFacetSearch.state;
+      const {values} = genericFacetSearch.state;
+      return {
+        ...genericFacetSearch.state,
+        values: values.map(({count, displayValue, rawValue}) => ({
```

Comment:
API always returns the path, so the path was part of the specific facet state which is completely illegal

Reply:
Yes, I'll add a test

Reply:
Yes please, my understanding is the loop is to exclude certain keys from being returned? A test for this would be useful,